### PR TITLE
Fixed onLoadPageDetails

### PR DESCRIPTION
**Pull request by ghost**
The onLoadPageDetails function got the mobileLayout always from a parentModel. Even if a mobileLayout was set.

### CODE REVIEW

File: src/EventListener/InheritMobileLayoutListener.php [hunk 1]
```diff
@@ -23,10 +23,12 @@ public function onLoadPageDetails(array $parentModels, PageModel $page): void
     {
         $page->mobileLayout = $page->includeLayout ? $page->mobileLayout : false;
 
-        foreach ($parentModels as $parentModel) {
-            if ($parentModel->includeLayout && $parentModel->mobileLayout) {
-                $page->mobileLayout = $parentModel->mobileLayout;
-                break;
+        if ($page->mobileLayout === false) {
```
Comment (leofeyer) [inline src/EventListener/InheritMobileLayoutListener.php]: I'd prefer an early return here:

```php
if ($page->mobileLayout !== false) {
    return;
}

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: src/EventListener/InheritMobileLayoutListener.php (hunk 1 above)
Comment (bohnmedia) [reply src/EventListener/InheritMobileLayoutListener.php]: Better.

**Comment by bohnmedia** [comment]
The code was changed. Is it ready to merge or is there anything I can do?

**Comment by bohnmedia** [comment]
I created a clean pull request with a single commit now.

https://github.com/contao/mobile-page-layout-bundle/pull/4

**Comment by leofeyer** [comment]
Why did you close the PR? It was ready to merge. 🤷‍♂

**Comment by bohnmedia** [comment]
Sorry, I was not sure if all the unnecessary commits would be merged too and wanted to make a clean pull request. Is the new one OK?

**Comment by leofeyer** [comment]
Sure. 👍

**Pull request closed without merging**